components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -171,19 +172,7 @@ private void insertAccessToken(String accessToken, String consumerKey, AccessTok
         }
 
         String sql;
-        if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
-            if (OAuth2ServiceComponentHolder.isConsentedTokenColumnEnabled()) {
-                sql = SQLQueries.INSERT_OAUTH2_ACCESS_TOKEN_WITH_IDP_NAME_WITH_CONSENTED_TOKEN;
-            } else {
-                sql = SQLQueries.INSERT_OAUTH2_ACCESS_TOKEN_WITH_IDP_NAME;
-            }
-        } else {
-            if (OAuth2ServiceComponentHolder.isConsentedTokenColumnEnabled()) {
-                sql = SQLQueries.INSERT_OAUTH2_ACCESS_TOKEN_WITH_CONSENTED_TOKEN;
-            } else {
-                sql = SQLQueries.INSERT_OAUTH2_ACCESS_TOKEN;
-            }
-        }
+        sql = getInsertAccessTokenQuery();
```

Comment:
Shall we declare and assign value in this line itself without line 174 since there is no if block now

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984/commits/59ba3e10b73d02dcf0d68de230df5add71a2ab97